docs/en/connector-v2/sink/Hive.md

```diff
@@ -100,6 +100,26 @@ Support writing Parquet INT96 from a timestamp, only valid for parquet files.
 
 Flag to decide whether to use overwrite mode when inserting data into Hive. If set to true, for non-partitioned tables, the existing data in the table will be deleted before inserting new data. For partitioned tables, the data in the relevant partition will be deleted before inserting new data.
 
+### schema_save_mode [enum]
+
+Before starting the synchronization task, different processing schemes are selected for the existing table structure on the target side.
+
+Option values:
+- `RECREATE_SCHEMA`: Will create when the table does not exist, delete and rebuild when the table exists
+- `CREATE_SCHEMA_WHEN_NOT_EXIST`: Will create when the table does not exist, skip when the table exists
+- `ERROR_WHEN_SCHEMA_NOT_EXIST`: Error will be reported when the table does not exist
+- `IGNORE`: Ignore the treatment of the table
+
+
+
+### table_format [string]
```

Comment:
Let's introduce `save_mode_create_template` instead of `table_format` and `partition_fields`
Please refer https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/sink/Doris.md#save_mode_create_template